docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -0,0 +1,212 @@
+PostgreSQL specific aggregation functions
+=========================================
+
+All of these functions are available from the ``django.contrib.postgres.aggregates``
```

Comment:
You should put a `.. module::` directive here. See https://github.com/django/django/blob/master/docs/ref/models/meta.txt#L5-L6 for an example.